packages/storefront/src/main.ts

```diff
@@ -40,10 +40,11 @@ Vue.config.ignoredElements = [/p-\w*/];
 Vue.use({
   install(vue: VueConstructor) {
     vue.component('A11yIcon', A11yIcon);
-    vue.component('ColorBadge', ColorBadge);
     vue.component('OpenBugTemplateInStackBlitz', OpenBugTemplateInStackBlitz);
     vue.component('PartialDocs', PartialDocs);
     // eslint-disable-next-line vue/multi-word-component-names
+    vue.component('Notification', Notification);
```

Comment:
Eslint annotation in previous line shouldn't be needed since `Notification` is a single word.

Reply:
True, but for unknown reasons it's complaining:
<img width="801" alt="Screenshot 2023-09-13 at 09 43 47" src="https://github.com/porsche-design-system/porsche-design-system/assets/30120450/fdafc898-baf9-43b1-8723-98d317dec67f">